### PR TITLE
Translations of Ruby 3-x release notes of March 30th 2023 (es)

### PR DESCRIPTION
**Pull request by vtamara**

### CODE REVIEW

File: es/news/_posts/2023-03-30-ruby-3-1-4-released.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+title: "Publicado Ruby 3.1.4 Released"
+author: "nagachika"
+translator: vtamara
+date: 2023-03-30 12:00:00 +0000
+lang: es
+---
+
+Se ha publicado Ruby 3.1.4.
+
+Ver detalles en la [publicación en GitHub](https://github.com/ruby/ruby/releases/tag/v3_1_4).
```
Comment (gfvcastro) [inline es/news/_posts/2023-03-30-ruby-3-1-4-released.md:12]:
```suggestion
Se ha publicado Ruby 3.1.4.

Esta versión incluye correcciones de seguridad.
Por favor vea detalles en los siguientes temas.

* [CVE-2023-28755: Vulnerabilidad ReDoS en URI]({%link es/news/_posts/2023-03-28-redos-in-uri-cve-2023-28755.md %})
* [CVE-2023-28756: Vulnerabilidad ReDoS en Time]({%link es/news/_posts/2023-03-30-redos-in-time-cve-2023-28756.md %})

Ver más detalles en la [publicación en GitHub](https://github.com/ruby/ruby/releases/tag/v3_1_4).
```

File: es/news/_posts/2023-03-30-ruby-3-2-2-released.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+lang: es
+---
+
+Se ha publicado Ruby 3.2.2.
+
+Esta versión incluye correcciones de seguridad.
+Por favor vea detalles en los siguientes temas.
+
+* [CVE-2023-28755: Vulnerabilidad ReDoS en URI]({%link en/news/_posts/2023-03-28-redos-in-uri-cve-2023-28755.md %})
+* [CVE-2023-28756: Vulnerabilidad ReDoS en Time]({%link en/news/_posts/2023-03-30-redos-in-time-cve-2023-28756.md %})
```
Comment (gfvcastro) [inline es/news/_posts/2023-03-30-ruby-3-2-2-released.md:16]:
```suggestion
* [CVE-2023-28755: Vulnerabilidad ReDoS en URI]({%link es/news/_posts/2023-03-28-redos-in-uri-cve-2023-28755.md %})
* [CVE-2023-28756: Vulnerabilidad ReDoS en Time]({%link es/news/_posts/2023-03-30-redos-in-time-cve-2023-28756.md %})
```

**Review by gfvcastro — CHANGES_REQUESTED** [review]
Thank you for your contribution!

File: es/news/_posts/2023-03-30-ruby-2-7-8-released.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+lang: es
+---
+
+Se ha publicado Ruby 2.7.8.
+
+Esta versión incluye correcciones de seguridad.
+Por favor revise detalles en los siguientes temas.
+
+* [CVE-2023-28755: Vulnerabilidad ReDoS en URI]({%link en/news/_posts/2023-03-28-redos-in-uri-cve-2023-28755.md %})
+* [CVE-2023-28756: Vulnerabilidad ReDoS en Time]({%link en/news/_posts/2023-03-30-redos-in-time-cve-2023-28756.md %})
```
[the remark this answers is not in the document]
Comment (vtamara) [reply es/news/_posts/2023-03-30-ruby-2-7-8-released.md:16]: Yes. Done. Thank you.

File: es/news/_posts/2023-03-30-ruby-3-0-6-released.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+lang: es
+---
+
+Se ha publicado Ruby 3.0.6.
+
+Esta versión incluye correcciones de seguridad.
+Por favor vea detalles en los siguientes temas.
+
+* [CVE-2023-28755: Vulnerabilidad ReDoS en URI]({%link en/news/_posts/2023-03-28-redos-in-uri-cve-2023-28755.md %})
+* [CVE-2023-28756: Vulnerabilidad ReDoS en Time]({%link en/news/_posts/2023-03-30-redos-in-time-cve-2023-28756.md %})
```
[the remark this answers is not in the document]
Comment (vtamara) [reply es/news/_posts/2023-03-30-ruby-3-0-6-released.md:16]: Yes. Done. Thank you.

File: es/news/_posts/2023-03-30-ruby-2-7-8-released.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+Esta versión también incluye correcciones a problemas de construcción.
+Ver más detalles en la [publicación en GitHub](https://github.com/ruby/ruby/releases/tag/v2_7_8).
+
+Después de esta versión, Ruby 2.7 llega a su EOL (Fin de vida).
```
[the remark this answers is not in the document]
Comment (vtamara) [reply es/news/_posts/2023-03-30-ruby-2-7-8-released.md:21]: Yes. Done. Thank you.

File: es/news/_posts/2023-03-30-ruby-3-0-6-released.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+Ruby 3.0, y Ruby 3.0 entra a la fase de seguridad.
+Esto significa que ya no retro-portaremos soluciones a fallas en Ruby 3.0
+excepto a fallas de seguridad.
+
+El periodo de la fase de mantenimiento de seguridad se ha programada a
+un año.
+Ruby 3.0 llegará a su EOL (fin de vida) y su soporte oficial terminará
+al final de la fase de mantenimiento.
+Por tanto, le recomendamos comenzar a planear la actualización a Ruby 3.1
+o 3.2
```
[the remark this answers is not in the document]
Comment (vtamara) [reply es/news/_posts/2023-03-30-ruby-3-0-6-released.md:31]: Yes. Done. Thank you.

File: es/news/_posts/2023-03-30-ruby-3-1-4-released.md (hunk 1 above)
Comment (vtamara) [reply es/news/_posts/2023-03-30-ruby-3-1-4-released.md:12]: The original post in english doesn't include that information, and according to the posts of the vulnerabilites Ruby 3.1.4 is not vulnerable (it is required just to update the gems uri and time).

File: es/news/_posts/2023-03-30-ruby-3-2-2-released.md (hunk 2 above)
Comment (vtamara) [reply es/news/_posts/2023-03-30-ruby-3-2-2-released.md:16]: Yes. Done. Thank you.

File: es/news/_posts/2023-03-30-ruby-3-1-4-released.md (hunk 1 above)
Comment (gfvcastro) [reply es/news/_posts/2023-03-30-ruby-3-1-4-released.md:12]: Hey @vtamara, from what I see, https://github.com/ruby/www.ruby-lang.org/blob/master/en/news/_posts/2023-03-30-ruby-3-1-4-released.md includes that information.

**Pull request merged**